src/components/custom/Workspace/Workspace.tsx

```diff
@@ -50,59 +51,84 @@ function Workspace({ ids }: { ids: string[] }) {
         }
     };
 
+    const handleCreateCrewClick = () => {
+        const params = new URLSearchParams(searchParams.toString());
+        params.set('create_mode', 'crew');
+        router.push(`/workspace/crew?${params.toString()}`);
+    };
+
+    // Type guard to check if it's a Crew
+    const isCrew = (item: Work | Crew): item is Crew => {
+        return (item as Crew).root_work !== undefined;
+    };
+
     return (
-        <div className="container mx-auto px-4 py-8 max-w-2xl">
-            <h1 className="text-3xl font-bold mb-6">{type} list</h1>
-            <Tabs
-                value={type}
-                onValueChange={value => handleTypeChange(value)}
-                className="w-full"
+        <div className="relative min-h-screen">
+            {/* "Create a Crew" button positioned at the top-left */}
+            <button
+                onClick={handleCreateCrewClick}
+                className="absolute top-4 left-4 w-96 h-10 bg-white text-black font-bold rounded-md shadow-md flex items-center justify-center space-x-2"
             >
-                <TabsList className="bg-zinc-900 border border-zinc-800">
-                    {[CREW, WORK].map(t => (
-                        <TabsTrigger
-                            key={t}
-                            value={t}
-                            className="data-[state=active]:bg-zinc-800"
-                        >
-                            {t}
-                        </TabsTrigger>
-                    ))}
-                </TabsList>
-            </Tabs>
+                <span className="text-black text-3xl font-bold">+</span>
+                <span className="font-bold">Create a Crew</span>
+            </button>
+
+            {/* Main Content */}
```

Comment:
yeah this main content will become middle panel, create crew button will go into left panel & right panel will be empty